ImGuizmo.cpp

```diff
@@ -224,7 +224,7 @@ namespace IMGUIZMO_NAMESPACE
       const vec_t& operator + () const { return (*this); }
       float Length() const { return sqrtf(x * x + y * y + z * z); };
       float LengthSq() const { return (x * x + y * y + z * z); };
-      vec_t Normalize() { (*this) *= (1.f / Length()); return (*this); }
+      vec_t Normalize() { (*this) *= (1.f / ( Length() ? Length() : FLT_EPSILON ) ); return (*this); }
```

Comment:
comparison is not correct. never compare a float to a literal value ( here == 0.f). Add epsilon of compare using difference.

Reply:
Thanks! After playing around with divisions whose divider was really close to 0, I see the impact of checking for equality with literal 0.f
Added a commit to rectify it, checking if result of `Length()` is greater than epsilon instead of different from 0.f